### ISSUE TITLE
inaccurate MANO annotation

### ISSUE THREAD

**Issue by ZhengdiYu** [lmb-freiburg/freihand#30]
This is the rendering results of 00005896.jpg of training set. Is this normal?
![image](https://user-images.githubusercontent.com/63605407/167585645-d9827feb-48a3-4e4f-839b-94007e4ff082.png)

**Comment by lixiny** [comment]
I  also witnessed a slightly offset on the mask/depth rendering, but it is not as much as your illustration. 
Here is my results.  ( 00005896.jpg )
![截屏2022-05-14 15 53 14](https://user-images.githubusercontent.com/32672902/168416542-1975cae8-463e-4c84-8bd2-4ddd4383e64b.png)
img, rendered depth from verts, depth + joints, masks.

**Comment by ZhengdiYu** [comment]
What kind of rendering method are you using? And what is your MANO model? I use (https://github.com/hassony2/manopth) to get a mesh in camera coordinate system by:

```
        uv_root, scale = mano[:, 58:60], mano[:, 60:]
        focal, pp = get_focal_pp(K)
        xyz_root = recover_root(uv_root, scale, focal, pp)

        mano_pose = torch.FloatTensor(pose).to(DEVICE)#.view(-1, 3)
        shape = torch.FloatTensor(shape).to(DEVICE)#.view(1, -1)

        assert mano_pose.size() == torch.Size((1, 48)), mano_pose.size()
        assert shape.size() == torch.Size((1, 10)), shape.size()
        output = mano_layer['r'](mano_pose, th_betas=shape) # already root-aligned in manolayer
        joints = output[1][0].cpu().detach().numpy()
        mesh = output[0][0].cpu().detach().numpy()

        trans = xyz_root[0] - joints[9]
        trans = trans[None]
        mesh = mesh + trans
        joints = joints + trans

        # milimeter
        #mesh = (output.vertices[0] * 1000).cpu().detach().numpy() # smplx
        mesh = (mesh * 1000)

**Comment by ZhengdiYu** [comment]
For most of cases, there are offsets. And of some of them, the offsets are acceptable:
![image](https://user-images.githubusercontent.com/63605407/168423039-d608bc94-1243-4f3a-bf14-ae23446b02a0.png)

![image](https://user-images.githubusercontent.com/63605407/168422638-2a91065e-090c-4b67-b74b-a38d60420034.png)
![image](https://user-images.githubusercontent.com/63605407/168422651-0eaea3ee-0fc5-466d-90af-2137ad140fbd.png)
![image](https://user-images.githubusercontent.com/63605407/168422684-f5b0d37c-1e06-4166-a5e7-86b42e02be99.png)
